.cirrus.yml

```
@@ -113,6 +114,16 @@ debian10_task:
     << : *RESOURCES_TEMPLATE
   << : *CI_TEMPLATE
 
+debian10_static_task:
+  container:
+    # Just uses a recent/common distro to run a static compile test.
+    # Debian 10 EOL: June 2024
+    dockerfile: ci/debian-10-static/Dockerfile
```

```suggestion
    dockerfile: ci/debian-10/Dockerfile
```
Guessing the existing image can be used for this or else forgot to add the new Dockerfile in the commit.

---

Thanks :).

I admittedly started this 2 days ago - and then it got late and I got distracted.